addons/hr_payroll/security/ir.model.access.csv

```diff
@@ -2,7 +2,7 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_hr_payroll_structure,hr.payroll.structure,model_hr_payroll_structure,base.group_hr_user,1,1,1,1
 access_hr_contribution_register,hr.contribution.register,model_hr_contribution_register,base.group_hr_user,1,1,1,1
 access_hr_salary_rule_category,hr.salary.rule.category,model_hr_salary_rule_category,base.group_hr_user,1,1,1,1
-access_hr_payslip,hr.payslip,model_hr_payslip,base.group_hr_user,1,1,1,1
+access_hr_payslip,hr.payslip,model_hr_payslip,base.group_user,1,1,1,1

```

Comment:
To try 
- add groups on field / div in view
- add groups on field directly if above does not work